cpu/stm32_common/include/periph_cpu_common.h

```diff
@@ -708,6 +708,168 @@ int dma_configure(dma_t dma, int chan, const volatile void *src, volatile void *
 #include "candev_stm32.h"
 #endif
 
+#ifdef MODULE_STM32_ETH
```

Comment:
This should go before the eth functions declaration, e.g. [here](https://github.com/RIOT-OS/RIOT/pull/10633/files#diff-569047d5a7bc79b2ac9e91264ac20eb0R851).

Reply:
Works just fine. Thanks :)